### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by millapihlajamaki**
Added a description file for the package, separated one of the functions (validate_user_response) in a separate file and wrote a test for that.

### CODE REVIEW

File: gonogo/R/gonogo.R [hunk 1]
```diff
@@ -4,22 +4,141 @@
...
+
+# IGNORE: screens?
+
+# Screen 1
+plot(NA, NA, xlim=c(0,1), ylim=c(0,1), xaxt="n", yaxt="n", xlab="", ylab="")
+text(x=0.5, y=0.5, labels="Welcome to the Go-No Go Task!", cex=1.5)
+
+# Screen 2: separate function for this?
+plot(NA, NA, xlim=c(0,1), ylim=c(0,1), xaxt="n", yaxt="n", xlab="", ylab="")
+text(x=0.5, y=0.5, labels="Instructions: In this task, you will see two types of stimuli, the letter A and the\nletter X. When you see the letter A, you should respond as quickly as\npossible by pressing the space bar. When you see the letter X, you should\nnot respond (do not press any key). You should try to be as fast and accurate\nas possible.",
```
Comment (EGabrilovica) [inline gonogo/R/gonogo.R]: the style of the code isn't the easiest to read here

**Comment by EGabrilovica** [comment]
Good descriptions! At least for the parts you've more or less finished, the descriptions are clear. Package/function descriptions look good to me.

File: gonogo/tests/testthat/test-validate_user_response.R [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  choice_unmatch <- random_letters[3]
+
+  result1 <- validate_user_response(choice = choice_match, choiceKeys = choice_keys)
+  result2 <- validate_user_response(choice = choice_unmatch, choiceKeys = choice_keys)
+
+  expect_false(result1[1] == result2[1])
+
+})
+
+
```
Comment (lucasrosenbusch) [inline gonogo/tests/testthat/test-validate_user_response.R:19]: Nice work on the testing. Makes sense to me!

File: gonogo/R/main.R [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+
+}
+
+print(output_example)
+
+
+
+
+
+
```
Comment (lucasrosenbusch) [inline gonogo/R/main.R:295]: Looks good to me! At this stage, the comments are very helpful.

**Pull request merged**